### PR TITLE
1.21

### PR DESCRIPTION
**Pull request by julian-smith-artifex-com**
Various minor fixes.

### CODE REVIEW

**Review by JorjMcKie — APPROVED** [review]

File: docs/installation.rst [hunk 1]
```diff
@@ -8,16 +8,28 @@ PyMuPDF should be installed using pip with::
...
   use the system package manager to install SWIG.
 
   * For example on Debian Linux, do: ``sudo apt install swig``
 
-* On Windows, install SWIG by following the instructions at:
-  https://swig.org/Doc4.0/Windows.html#Windows_installation.
+* On Windows:
+
+  * Install Visual Studio 2019. If not installed in a standard location, set
+    environmental variable `PYMUPDF_SETUP_DEVENV` to the location of the
```
Comment (jamie-lemon) [inline docs/installation.rst:28]: Hmmm,  it is a bit of a grey area, but double backpacks for anything `code related` would be my preference.

File: docs/installation.rst [hunk 2]
```diff
@@ -8,16 +8,28 @@ PyMuPDF should be installed using pip with::
...
+    `devenv.com` binary.
+
+  * Install SWIG by following the instructions at:
+    https://swig.org/Doc4.0/Windows.html#Windows_installation.
 
 * On MacOS, install MacPorts using the instructions at:
   https://www.macports.org/install.php
```
Comment (jamie-lemon) [inline docs/installation.rst:35]: See previous comment

File: tests/README.md [hunk 3]
```diff
@@ -79,3 +79,13 @@ TOTAL                                                      6714   1422    79%
...
+
+## Known test failure with non-default build of MuPDF
+
+If PyMuPDF has been built with a non-default build of MuPDF (using
+environmental variable `PYMUPDF_SETUP_MUPDF_BUILD`), it is possible that
+`tests/test_textbox.py:test_textbox3()` will fail, because it relies on MuPDF
+having been built with PyMuPDF's customized configuration, `fitz/_config.h`.
+
+One can skip this particular test by adding `-k 'not test_textbox3'` to the
+pytest command line.
```
Comment (jamie-lemon) [inline tests/README.md:91]: Double back ticks for all the code refs?

**Review by jamie-lemon — CHANGES_REQUESTED** [review]
Just a few ideas to tidy things up if possible.

**Review by jamie-lemon — APPROVED** [review]

**Pull request merged**